app/Repositories/ClientRepository.php

```diff
@@ -14,9 +16,18 @@ public function __construct()
         parent::__construct(Client::class);
     }
 
+    public function createBaseQuery(): Builder
+    {
+        return $this->model->where(function(Builder $query) {
+            return $query
+                ->whereYear('arrival_date', '=', Carbon::now()->year)
```

Comment:
```suggestion
                ->whereYear('arrival_date', '>=', Carbon::now()->year - 1)
```
Looking for arrival date newer than last year seems better idea.

Reply:
This should never be a relevant case, we assume end of year as time with no guests, otherwise however we approach it we will end with edge cases (e.g. someone arrived on 31st and and it's 1st, so suddenly they're no longer shown) - if it beccomes a relevant case, a more proper handling like an `or arrival_date up to two weeks prior` or something like that will have to be implemented, and that will be a separate MR.